### ISSUE TITLE
Getting build failures on 1.0.0

### ISSUE THREAD

**Issue by js** [bitrise-steplib/steps-npm#12]
All my builds started failing on this step with no other configuration changes, then I realised it was set to "latest" and there was a new version. Reverted to 0.9.1 and everything builds as usual again.

Here the raw log entry from the failing step, not a lot of info to go by unfortunately:
```
+------------------------------------------------------------------------------+
| (7) Install apollo-codegen                                                   |
+------------------------------------------------------------------------------+
| id: npm                                                                      |
| version: 1.0.0                                                               |
| collection: https://github.com/bitrise-io/bitrise-steplib.git                |
| toolkit: go                                                                  |
| time: 2018-11-22T04:54:24-08:00                                              |
+------------------------------------------------------------------------------+
|                                                                              |
[34;1mConfig:
[0m- Workdir: /Users/vagrant/git
- Command: install -g apollo@1.9.2
- NpmVersion: 

[34;1mAutodetecting npm version[0m
Checking package.json for npm version
[33;1mNo npm version found in package.json[0m
Locating system installed npm
npm found at /usr/local/bin/npm
[32;1m$ npm "--version"[0m
6.4.1

[34;1mRunning user provided command[0m
[32;1m$ npm "install -g apollo@1.9.2"[0m
[31;1mError running command install -g apollo@1.9.2: error running npm command: exit status 1[0m

|                                                                              |
+---+---------------------------------------------------------------+----------+
| [31;1mx[0m | [31;1mInstall apollo-codegen (exit code: 1)[0m                         | 5.32 sec |
+---+---------------------------------------------------------------+----------+
| Issue tracker: https://github.com/bitrise-steplib/steps-npm/issues           |
| Source: https://github.com/bitrise-steplib/steps-npm                         |
+---+---------------------------------------------------------------+----------+
```

**Comment by bitce** [comment]
Hi @js, thanks for reporting! We are aware of the issue and the fix will soon be released, the same goes for the error raised on  #8

**Comment by bitce** [comment]
That's correct, didn't mean to imply this at all, it was just an issue within the same release.

**Comment by bitce** [comment]
A new step version was released that fixed the issue, thanks again!

**Comment by lszucs** [comment]
Hey @js,

As stated, we have just published version 1.0.1, which fixes issues with npm commands made up of several components.

For more info, see: https://github.com/bitrise-steplib/steps-npm/releases/tag/1.0.1

For now, I'll close this issue, but if you need more info/help feel free to open a new issue!
